add the specificity of the :host() pseudo-class (#31549)
mdn/content: files/en-us/web/css/_colon_host_function/index.md

> **Note:** This has no effect when used outside a shadow DOM.

> > **Note:** Unlike similar pseudo-classes, like [:is()](/en-US/docs/Web/CSS/:is) or [:not()](/en-US/docs/Web/CSS/:not), the specificity of **`:host()`** is that of a pseudo-class, plus the specificity of its argument.

Reviewer: ```suggestion
> **Note:** Unlike similar pseudo-classes, like {{CSSxRef(":is", ":is()")}} or {{CSSxRef(":not", ":not()")}}, the specificity of **`:host()`** is that of a pseudo-class, plus the specificity of its argument.
```

use macros is better

Reviewer 2: - We now have two Note texts one after the other. The first note text can actually be combined with the first paragraph (edited here slightly), like so:
"The **`:host()`** [CSS](/en-US/docs/Web/CSS) [pseudo-class](/en-US/docs/Web/CSS/Pseudo-classes) function selects the shadow host of the [shadow DOM](/en-US/docs/Web/API/Web_components/Using_shadow_DOM) containing the CSS it is used inside, but only if the function's parameter matches the shadow host. This allows the selection of a custom element from inside its shadow DOM. `:host()` has no effect when used outside a shadow DOM."

- This note can be slightly tweaked:
```suggestion
> **Note:** Unlike similar pseudo-classes such as {{CSSxRef(":is", ":is()")}} and {{CSSxRef(":not", ":not()")}}, which only take the specificity of their argument, the specificity of **`:host()`** is that of a pseudo-class **_plus_** the specificity of its argument.
```

Reviewer 3: https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes doesn't have `:host()` listed in the list of functional pseudo-classes. It should.

That said, i would recommend replacing "similar" with "similar functional" so we know why they're similar (because they are otherwise not really similar at all)
```suggestion
> **Note:** Unlike other functional pseudo-classes, like [:is()](/en-US/docs/Web/CSS/:is) or [:not()](/en-US/docs/Web/CSS/:not), the specificity of **`:host()`** is both that of the pseudo-class plus the specificity of its argument.
```

Reviewer 2: Thanks @estelle for your valuable input.
We can split up this info in the note so that it doesn't get confusing and the differences are more apparent.

```suggestion
> **Note:** While other functional pseudo-classes such as {{CSSxRef(":is", ":is()")}} and {{CSSxRef(":not", ":not()")}} accept a list of selectors as their parameters, `:host()` takes a single compound selector as its parameter. In addition, while `:is()` and `:not()` only take into account the specificity of their argument, the specificity of `:host()` is both the specificity of the pseudo-class **and** the specificity of its argument.
```
